app/views/draftmetadata/draftMetadataChecksWithErrorDownload.scala.html

```diff
@@ -8,10 +8,12 @@
         <div class="govuk-grid-column-two-thirds">
             @draftMetadataChecksActionProcess(actionMessage, detailsMessage)
 
-            <p class="govuk-body">The report below contains details about issues found.</p>
+            <p class="govuk-body">The report below contains detailed errors with reference to the file path and column title, which caused the error.</p>
 
             @downloadMetadataLink(consignmentId, "Download report", routes.DraftMetadataChecksResultsController.downloadErrorReport(consignmentId).url)
-            <p class="govuk-body">Once you have addressed this issue upload a revised metadata file.</p>
+            <p class="govuk-body">Once you have addressed the errors, you can re-upload the metadata file.</p>
+
+            @daAlert(heading = "Leaving and returning to this transfer", content = Html(s"You can sign out and return to continue working on this transfer at any time from <a class='govuk-link' href='${routes.ViewTransfersController.viewConsignments(1)}'>View transfers</a>."))
```

Comment:
Is that the correct input for the route with the `1`, ie: `... ViewTransfersController.viewConsignments(1)`

Reply:
Yep, that would be the first page of the view transfers page. The same is done in the other views that also have a view transfer link.